pkg/storage/async_store.go

```diff
@@ -21,23 +22,28 @@ type IngesterQuerier interface {
 	GetChunkIDs(ctx context.Context, from, through model.Time, matchers ...*labels.Matcher) ([]string, error)
 }
 
+type AsyncStoreCfg struct {
+	IngesterQuerier      IngesterQuerier
+	QueryIngestersWithin time.Duration
```

Comment:
nit: I think it would really be helpful to add doc string about what this `QueryIngestersWithin` config does exactly.

Reply:
Thanks for the review! I have taken care of the suggestion.